server/src/main.rs

```diff
@@ -60,32 +62,32 @@ fn run_server(host: &str, port: &str, repo_root: &Path) {
 }
 
 fn get_repo(req: &HttpRequest<AppState>) -> impl Responder {
```

Comment:
I think we should have a corresponding test for this to verify the status codes come back as expected.

Reply:
Should I wait for Javier's test PR #28 to land then add more tests to that?